words2num/lang_EN_US.py

```diff
@@ -55,7 +56,7 @@
     'octodecillion': (10**57, 'X'),
     'novemdecillion': (10**60, 'X'),
     'vigintillion': (10**63, 'X'),
-    'centillion': (10**303, 'X'),
+    'centillion': (10**303, 'X')
```

Comment:
curious what you think of this convention

Reply:
Purely personal preference to not have dangling commas